.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "Src/RE2.Native/re2"]
 	path = src/RE2.Native/re2
-	url = https://github.com/google/re2.git
+	url = https://github.com/google/re2.git
```

Comment:
Hello, @thatsagreatidea, I'm not sure that we should do that. What we've recently preferred is to simply check in an optimized release build of RE2.dll. Would that work for your purposes?

Reply:
yes, thank you